apps/console/src/features/connections/components/edit/forms/authenticators/email-otp-authenticator-form.tsx

```diff
@@ -97,9 +97,9 @@ interface EmailOTPAuthenticatorFormInitialValuesInterface {
      */
     EmailOTP_OTPLength: string;
     /**
-     * Allow OTP token to have 0-9 characters only.
+     * Allow OTP token to have mix of 0-9 and A-Z characters.
      */
-    EmailOTP_OtpRegex_UseNumericChars: boolean;
+    EmailOTP_OtpRegex_UseAlphanumericChars: boolean;
```

Comment:
`OtpRegex` feels redundant. If we can fix this, let's add that as well

Reply:
Fixed with 2d27580c0faa32a8c2fa5df319a653fd259ec166